TestAssets/TestProjects/PerformanceTestProjects/SingleTargetGraph/SingleTargetP0/Program.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
```

Comment:
Yes, you're right.  I missed that.  They only have a project-to-project dependency though.  No actual code dependency.  Shouldn't we be testing a real scenario with actual code dependency too?  These are just a bunch of empty mains with no dependencies.